src/screens/timeStamping/EventEdition/styles.ts

```diff
@@ -15,6 +21,18 @@ export default StyleSheet.create({
     paddingHorizontal: PADDING.MD,
     height: HEADER_HEIGHT,
   },
+  billedHeader: {
+    flexDirection: 'row',
+    backgroundColor: COPPER[400],
+    alignItems: 'center',
+    height: BILLED_HEADER_HEIGHT,
+  },
+  billedHeaderText: {
+    ...FIRA_SANS_REGULAR.MD,
+    color: WHITE,
+    textAlign: 'center',
+    flex: 1,
```

Comment:
si tu enleves la View dans le code, tu peux faire les changements suivants : 
- enlever `flex: 1`
- mettre `backgroundColor` dans le style de la balise `Text`
- utiliser un `padding` (PADDING.MD par exemple) plutot que `height` (a noter dans un coin de ta tete ou ton carnet, je t'ai fait ce retour plusieurs fois dans le sprint : on définit très rarement la hauteur stricte d'un element. on utilise plutot les marges et le padding pour les positions / espacements)
- tu peux supprimer `billedHeader`

Reply:
Yes c'est noté :)